wolfssl/wolfcrypt/port/Renesas/renesas-tsip-crypt.h

```diff
@@ -202,15 +108,305 @@ typedef wolfssl_TSIP_Hash wc_Sha;
     typedef struct {
         tsip_aes_key_index_t tsip_keyIdx;
         word32               keySize;
+        byte                 setup;
     } TSIP_AES_CTX;
+#endif 
+
+
+
+
+/****
+typedef struct user_PKCbInfo {
+    uint32_t user_key_id;
+} user_PKCbInfo;
+***/
+/*
+ * structure for ECDH key exchange
+ */
+typedef struct TsipUserCtx {
+    
+    uint32_t user_key_id;
+//    user_PKCbInfo *user_Cbinfo;
```

Comment:
Looks like this one-line comment should be deleted, about `user_Cbinfo`

Reply:
Deleted it.